run/src/run.ts

```diff
@@ -6,16 +6,14 @@ import {
   MasterWrapper,
   Subscription,
   Engine,
+  ApiFactory,
 } from './types';
 import { multicastNow } from './multicastNow';
 
 let currentId = 0;
 
 function cuid(): number {
-  if (currentId === ++currentId) {
-    currentId = 0;
-  }
-  return currentId;
+  return currentId++;
```

Comment:
Asking just to clarify: red code was doing something about integer overflow, and green code ignores it?

Reply:
I am simply not sure if this is needed. do we expect people to have more than 2^32 events?

Reply:
I've added it back in, better be sure